packages/server/src/workers/cron.ts

```diff
@@ -41,11 +41,12 @@ let worker: Worker<CronJobData> | undefined = undefined;
  * Initializes the Cron worker.
  * Sets up the BullMQ job queue.
  * Sets up the BullMQ worker.
- * @param config The Redis config.
+ * @param redisConfig The Redis config.
+ * @param bullmqConfig The BullMQ config.
  */
-export function initCronWorker(config: MedplumRedisConfig): void {
+export function initCronWorker(redisConfig: MedplumRedisConfig, bullmqConfig?: MedplumBullmqConfig): void {
```

Comment:
Thoughts on changing this to `config: MedplumServerConfig`, which would include both `redisConfig` and `bullmqConfig`?

Reply:
Yeah I almost did that, took like 10 minutes trying to decide between the two signatures.

I think we should do that now that I got your approval 👍